modules/ROOT/pages/create-outbound-message-flow.adoc

```diff
@@ -1,126 +1,139 @@
-The reference ID is added to the message flow.
+The source message type in an outbound message flow defines the structure of the messages that Partner Manager sends to a partner, either directly or through a 3rd party connection. Partner Manager supports the JSON and XML source message types.
+
+To create the source message type:
+
+. In the message flow, expand the *Source message type* section and click *Select*.
+. In the *Source message type* window, do either of the following:
+* Select an existing source message type and click *Select*.
+* Click *New* to create a new source message type, as described in <<xref:partner-manager-create-message-type,Create and Configure Message Types>>.
 
+[[reference-ids]]
+== Configure Reference IDs for the Partner and Host
+
+Partner Manager uses the partner reference ID and host reference ID to locate the appropriate message flow for routing outbound messages. For information about about how this routing process works, see xref:partner-manager-identifiers[Sender and Receiver Identifiers] and xref:outbound-message-routing.adoc[Outbound Message Routing].
+
+To configure a partner reference ID, either select *All available identifiers* or select a specific partner identifier.
+
+To configure a host reference ID, either select *Ignore* or select your organization's internal reference ID (such as the business unit name) as it appears in the data element of the source JSON or XML message from your backend systems.
+
+[[translation-map]]
 == Import the Translation Map
 
-=== Prerequisites
-To create your outbound EDI-to-application message (JSON or XML) map in Anypoint Studio, see xref:partner-manager-maps.adoc[EDI Translation Maps].
+A translation map in an outbound message flow transforms outbound messages from your organization's enterprise application format to the format your partners expect. Create a translation map by using DataWeave. For more information, see see xref:partner-manager-maps.adoc[EDI Translation Maps].
 
-=== Steps
+To import the translation map:
 
 . In the message flow, expand the *Map* section.
-. Click *Import* and then click *Choose file*.
-. In the *New mapping file* window, navigate to the DataWeave map you want to use and then click *Open*.
-. Click *Upload mapping*. +
+. Click *Import < Choose file*.
+. Navigate to the DataWeave map you want to use and click *Open*.
+. Click *Upload mapping*.
+
 The translation map is added to the message flow.
 
+[[send-to-partner-message-type]]
 == Configure the Target Message Type
 
+The target message type in an outbound message flow defines the structure of the messages that Partner Manager sends to the partner, either directly or via a 3rd party connection. Partner Manager supports the AS2, FTP, and SFTP formats for send to message types.
```

Comment:
"send to message types" = "target message types"?
If you STET, then hyphenate: "send-to message types" or "send-to-message types," as appropriate.